### PR TITLE
Miscellaneous code quality improvements

### PR DESCRIPTION
**Pull request by ReperakDev**
Hello. Here's a summary of this PR:

* Fix overuse of static properties/methods
* Use a more Java-like programming style (`this` keyword, accessing via getters)
* Better use of Lombok
* Dedicated method for loading Config (constructors are bad with exceptions)
* Better exception types and handling
* Rename the main class to "ZtereoMUSIC"

### CODE REVIEW

**Review by ZtereoHYPE — COMMENTED** [review]
ZtereoMUSIC was supposed to be a temporary name hence why i left it to Bot for now... Ill still accept it though

**Comment by ZtereoHYPE** [comment]
care to explain why constructors are bad

File: src/main/java/codes/ztereohype/ztereomusic/database/Config.java [hunk 1]
```diff
@@ -3,22 +3,29 @@
...
 
 import java.io.File;
+import java.io.FileNotFoundException;
 import java.util.HashMap;
 import java.util.Map;
 
 public class Config {
-    private @Getter final Map<String, String> propreties = new HashMap<>();
-    private final String path;
+    private @Getter Map<String, String> propreties = new HashMap<>();
```
Comment (ZtereoHYPE) [inline src/main/java/codes/ztereohype/ztereomusic/database/Config.java:14]: lol you didnt change the typo in the end

**Comment by ReperakDev** [comment]
> care to explain why constructors are bad

Constructors can't throw exceptions and generally it's dirty to have a constructor run compute code.

**Comment by ZtereoHYPE** [comment]
this is good i like

**Comment by ZtereoHYPE** [comment]
Fuck, i closed instead of merging.

**Pull request merged**